arctic/store/_ndarray_store.py

```diff
@@ -163,6 +158,41 @@ def get_info(self, version):
         ret['rows'] = int(version['up_to'])
         return ret
 
+    def _chunked_read(self, arctic_lib, version, symbol, date_range, read_preference=None):
```

Comment:
I can of course bring all this code back into the pandas storage class. I'm not even sure why its in the ndarray storage class since the majority of it seems to be mongo/schema specific and not really related to pandas, numpy, etc.